ResearchKit/Common/ORKImageCaptureView.m

```diff
@@ -140,8 +140,8 @@ - (void)setError:(NSError * __nullable)error {
     _error = error;
     _headerView.alpha = error==nil ? 0 : 1;
     _headerView.instructionLabel.text = [error.userInfo valueForKey:NSLocalizedDescriptionKey];
-    if (error && self.continueTitle) {
-        _continueSkipContainer.continueButtonItem.title = self.continueTitle;
+    if (error) {
+        _continueSkipContainer.continueButtonItem = nil;
         _continueSkipContainer.skipButtonItem = nil;
     }
```

Comment:
Shouldn't the other case should be handled here, i.e. restoring the correct continue button item if error -> nil?